streamlit-snowpark-dynamic-filters/.streamlit/secrets.toml

```diff
@@ -0,0 +1,9 @@
+[snowpark]
+user = "test"
+password = "test"
+account = "test"
+warehouse = "test"
+database = "STREAMLIT_DEMO"
```

Comment:
This isn't the default database, right? I thought it was `TEST`

Reply:
Yeah, this was provided in the docs

Reply:
Okay, if it starts up with database `STREAMLIT_DEMO` I don't think there's anything to worry about - otherwise if it's still `TEST` by default, we might need to manually create `CREATE DATABASE "STREAMLIT_DEMO"`